### ISSUE TITLE
Unspecific error running full disk test on SD card

### ISSUE THREAD

**Issue by c0xc** [c0xc/CapacityTester#17]
When testing an SD card fails, the read verification step failed "after 0 GB":

![CT_SD_false_error_Screenshot_20231016_104301](https://github.com/c0xc/CapacityTester/assets/7249348/280b7e34-220f-4144-b8b4-1bd4788c7ce7)

This SD card should be ok though, so a more specific error might be useful.

**Comment by c0xc** [comment]
It turns out, the USB adapter that was used for this test is defective and causes random data corruption, even when copying files (to any SD card). So this is not a bug in the program at all, but it's not something you'd expect either.

Not that it would matter, but it's this type of USB adapter that stopped working:
![tecknet_61fDoMUPdjL_AC_SL1500_](https://github.com/c0xc/CapacityTester/assets/7249348/9640feb4-35b1-4395-a3df-63e4e77e4b89)

Because the adapter caused random corruption, CapacityTester showed random values for the real (working) capacity, like 476 MB in the case mentioned above.

CapacityTester has no way of knowing that the USB adapter damages data. It would be possible to display whether a read error (from the operating system) has been reported but usually, no such error is reported. 

To sum it up, it's not a bug, it worked as expected (and I now have to erase or paint over the red "x" I wrote on the card). Maybe this "issue" is useful for someone.
